thunder/core/trace.py

```diff
@@ -371,6 +371,7 @@ def keyfn(class_or_module: type | ModuleType) -> str:
 
             if include_decorators:
                 program.append("from thunder.executors.torchex import no_autocast")
+                program.append("from thunder.core.decorators import avoid_torch_nccl_record_streams")
```

Comment:
Probably it's time to wrap Thunder-generated functions with just "`default_decorator`" that would hide all of this.